llvm/lib/Analysis/IVDescriptors.cpp

```diff
@@ -1063,7 +1063,7 @@ unsigned RecurrenceDescriptor::getOpcode(RecurKind Kind) {
 SmallVector<Instruction *, 4>
 RecurrenceDescriptor::getReductionOpChain(PHINode *Phi, Loop *L) const {
   SmallVector<Instruction *, 4> ReductionOperations;
-  unsigned RedOp = getOpcode(Kind);
+  unsigned RedOp = getOpcode();
```

Comment:
Are there any users remaining that take kind? If not can it be removed?

Reply:
Not sure what you mean: Kind is a private member of the RecurrenceDescriptor class.